project/Dependencies.scala

```diff
@@ -216,6 +216,13 @@ object Dependencies {
     )
   )
 
+  val Reference = Seq(
+    // connector specific library dependencies and resolver settings
+    libraryDependencies ++= Seq(
+      "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.0" // BSD-style
```

Comment:
another quick question: so every connector should include this ?

Reply:
If it needs help converting between Java and Scala types (collections, futures, durations) - so most often, yes.

Reply:
But the java8 compat library comes as a transitive dependency to akka-actor. Therefore we should not add it explicitly here and instead use the transitively resolved one.

That will also make sure that the right version of java8-compat is used.

I'll remove it from here.